src/site/content/en/blog/deprecating-excalidraw-electron/index.md

```diff
@@ -0,0 +1,267 @@
+[Windows installers](https://www.electronjs.org/docs/api/auto-updater#windows). Turns out, some of
+the promised features need a detailed look at the small print.
+
+- For example, automatic updates _"are [currently] only [supported] on macOS and Windows. There is
+  no built-in support for auto-updater on Linux, so it is recommended to use the distribution's
+  package manager to update your app"_.
+
+- Developers can create native menus by calling `Menu.setApplicationMenu(menu)`. On Windows and
+  Linux, the menu will be set as each window's top menu, while on macOS there are many
+  system-defined standard menus, like the
+  [Services](https://developer.apple.com/documentation/appkit/nsapplication/1428608-servicesmenu?language=objc)
+  menu. To make one's menus a standard menu, developers should set their menu's `role` accordingly,
+  and Electron will recognize them and make them become standard menus. This means that a lot of
+  menu-related code will make use of the following platform check:
+  `const isMac = process.platform === 'darwin'`.
+
+- Windows installers can be made with
+  [windows-installer](https://github.com/electron/windows-installer). The README of the project
+  highlights that _"for a production app you need to sign your application. Internet Explorer's
+  SmartScreen filter will block your app from being downloaded, and many anti-virus vendors will
+  consider your app as malware unless you obtain a valid cert"_.
+
+Looking at just these three examples, it is clear that Electron is far from "write once, run
+everywhere". Distributing an app on app stores requires
+[code signing](https://www.electronjs.org/docs/tutorial/code-signing), a security technology for
+certifying app ownership. Packaging an app requires using tools like
+[electron-forge](https://github.com/electron-userland/electron-forge) and thinking about where to
+host packages for app updates. It gets complex relatively quickly, especially when the objective
+truly is cross platform support. I want to note that it is _abolutely_ possible to create stunning
+Electron apps with enough effort and dedication. For Excalidraw Desktop, we were not there.
+
+## Where Excalidraw Desktop left off
+
+Excalidraw Desktop so far is basically the Excalidraw Web app bundled as an
+[`.asar`](https://github.com/electron/asar) file with an added **About Excalidraw** window. The look
+and feel of the application is almost identical to the Web version.
+
+<figure class="w-figure">
+  <img src="excalidraw-desktop.png" alt="The Excalidraw Desktop application running in an Electron wrapper.">
+  <figcaption class="w-figcaption">Excalidraw Desktop is almost indistinguishable from the Web version</figcaption>
+</figure>
+
+<figure class="w-figure">
+  <img src="about-excalidraw.png" width="400" alt="The Excalidraw Desktop 'About' window displaying the version of the Electron wrapper and the Web app.">
```

Comment:
@robdodson This width will obviously cause overflowing on smaller screens. What's the preferred way of making sure an image takes up all of the available screen space?

Reply:
We deal with this in our CSS by having a rule `max-width: 100%`.

<img width="953" alt="Screen Shot 2020-12-18 at 12 02 58" src="https://user-images.githubusercontent.com/145676/102607722-0f3c3280-4129-11eb-8ff8-691a0413a4a9.png">